.env

```diff
@@ -0,0 +1,2 @@
+NETWORK=testnet3
+PRIVATE_KEY=APrivateKey1zkp5D76xqTBnu2CVUfxquE5gYfkM7TybYn7A7jWs8wUjdnB
```

Comment:
should not submit sensitive information like private key.

Reply:
Fixed